.github/workflows/docker-cli-main.yaml

```diff
@@ -2,10 +2,7 @@ name: Docker CLI Deployment
 on:
   push:
     branches:
-      - main
       - develop
-    tags:
-      - v*
```

Comment:
This won't change the tag that it publishes to for develop. `.github/scripts/planCliBuild.sh` needs updating to tag it as latest.